src/bitgo/keyutil.js

```diff
@@ -0,0 +1,44 @@
+var ecurve = require('ecurve')
+var curve = ecurve.getCurveByName('secp256k1')
+const BigInteger = require('bigi')
+const ECPair = require('../ecpair')
+
+/**
+ * Create an ECPair from the raw private key bytes
+ * @param {Buffer} buffer - Private key for the ECPair. Must be exactly 32 bytes.
+ * @param {Object} [network] - Network for the ECPair. Defaults to bitcoin.
+ * @return {ECPair}
+ */
+function privateKeyBufferToECPair (buffer, network) {
+  if (!Buffer.isBuffer(buffer) || buffer.length !== 32) {
+    throw new Error('invalid private key buffer')
+  }
+
+  var d = BigInteger.fromBuffer(buffer)
+
+  if (d.signum() <= 0 || d.compareTo(curve.n) >= 0) {
+    throw new Error('private key out of range')
+  }
+
+  return new ECPair(d, null, { network })
+}
+
+/**
+ * Get the private key as a 32 bytes buffer. If it is smaller than 32 bytes, pad it with zeros
+ * @param {ECPair} ecPair
+ * @return Buffer - 32
+ */
+function privateKeyBufferFromECPair (ecPair) {
+  if (!(ecPair instanceof ECPair)) {
+    throw new TypeError(`invalid argument ecpair`)
+  }
+
+  if (!ecPair.d) throw new Error('Missing private key')
+
+  return ecPair.d.toBuffer(32)
```

Comment:
It would be easier to review, if moving and amending code were done as two separate steps. 

It seems that this is the remainder of the deprecated `ECPair.prototype.getPrivateKeyBuffer` function making use of an existing function that gets a buffer directly from the private key in the ECPair.

Reply:
I'll split it up the PRs to make it clearer

Reply:
Thanks